index.html

```diff
@@ -1,12 +1,27 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css" /> 
-    <title>Document</title>
-</head>
-<body>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" type="text/css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+    />
     <script src="script.js"></script>
-</body>
-</html>
\ No newline at end of file
+    <title>John Doe's Personal Website</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="navbar">
+          <div class="logo">
+            <li><a href="/">John Doe</a></li>
+          </div>
+          <div class="nav-links">
+            <li><a href="#projects">Projects</a></li>
```

Comment:
So the best practice for a navigation would be the following structure

```
<nav>
  <ul>
    <li><a></a></li>
    <li><a></a></li>
  </ul>
</nav>
```

Reply:
you could add another container in it if needing for spacing or flex box or something like that but the general structure is best like this, especially for screen readers